src/app/zap-templates/zcl/data-model/chip/temperature-control-cluster.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <struct name="TemperatureLevelStruct">
+    <cluster code="0x0056"/>
+    <item fieldId="0" name="Label" type="char_string" length="64" isNullable="false" writable="false"/>
+    <item fieldId="1" name="TempLevel" type="int8u" isNullable="false" writable="false"/>
```

Comment:
I missed this, but this does not match the spec....  I guess I'll fix it up in a separate PR, but @jadhavrohit924 please be more careful in the future.

Reply:
https://github.com/project-chip/connectedhomeip/pull/26691

Reply:
@jadhavrohit924 Please take a note